src/Components/Reservation/ReservationForm.jsx

```diff
@@ -0,0 +1,198 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { ColorRing } from 'react-loader-spinner';
+import { useParams, useNavigate } from 'react-router-dom';
+import { fetchCars } from '../../Redux/Car/carSlice';
+import {
+  addReservation,
+  resetState,
+} from '../../Redux/Reservation/reservationSlice';
+import '../../index.css';
+
+const ReservationForm = () => {
+  function getCurrentDate() {
```

Comment:
- [ ] Kindly, convert this function to ES6, as requested in the previous review.  

Refactoring to ES6 is easy:

```javascript
const getCurrentDate= () => {
  // ...
}
```

Don't worry about the nesting.  This is how it's done in the industry (e.g. no "function" keyword at all).

That way, your code will comply with the standard. 👍

Reply:
fixed, thank you for noting it out